apiv2/db_import/tests/test_db_dataset_import.py

```diff
@@ -74,74 +76,82 @@ def expected_funding_sources() -> list[dict[str, Any]]:
 def test_import_dataset_new(
     verify_dataset_import: Callable[[list[str]], models.Dataset],
 ) -> None:
-    actual = verify_dataset_import([])
+    actual = verify_dataset_import()
     assert len(actual.authors) == 0
     assert len(actual.funding_sources) == 0
 
 
 # Tests update of dataset existing in database
 def test_import_dataset_update(
+    sync_db_session: Session,
     verify_dataset_import: Callable[[list[str]], models.Dataset],
 ) -> None:
-    populate_dataset()
-    populate_dataset_authors()
-    populate_dataset_funding()
-    actual = verify_dataset_import([])
+    populate_dataset(sync_db_session)
+    populate_dataset_authors(sync_db_session)
+    populate_dataset_funding(sync_db_session)
+    sync_db_session.commit()
+    actual = verify_dataset_import()
     assert len(actual.authors) == 1
     assert len(actual.funding_sources) == 1
 
 
 # Tests addition of new authors, updating entries already existing in db, and removing stale authors
```

Comment:
pytest will the show the docstring for the test if it has one. Consider moving these comments to the docstring.